forc/src/utils/defaults.rs

```diff
@@ -49,26 +49,64 @@ path = "tests/harness.rs"
 }
 
 pub(crate) fn default_program() -> String {
-    r#"script;
+    r#"contract ;
 
-fn main() {
+    abi MyContract {
 
-}
-"#
-    .into()
-}
+    }
 
-pub(crate) fn default_test_program() -> String {
-    r#"
+    impl MyContract for Contract {
 
-#[tokio::test]
-async fn harness() {
-    assert_eq!(true, true);
-}
+    }
 "#
     .into()
 }
 
+pub(crate) fn default_test_program(project_name: &str) -> String {
+    format!(
+        "{}{}{}{}{}",
+        r#"
+    use fuel_tx::{ContractId, Salt};
+    use fuels_abigen_macro::abigen;
+    use fuels_contract::{contract::Contract, parameters::TxParameters};
+    use fuels_signers::util::test_helpers;
+    
+    // Load abi from json
+    abigen!(MyContract, "./"#,
+        project_name,
+        r#".json");
+    
+    async fn get_contract_instance() -> (MyContract, ContractId) {
+    
+        // Build the contract
```

Comment:
Since the contract is already built, rather than "build the contract" would prefer something like "deploy the compiled contract"?

Reply:
Changed comment to be more correct